jib-gradle-plugin/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 
 - Glob pattern support for `jib.extraDirectories.permissions`. ([#1200](https://github.com/GoogleContainerTools/jib/issues/1200))
 - Support for image references with both a tag and a digest. ([#1481](https://github.com/GoogleContainerTools/jib/issues/1481))
+- The `DOCKER_CONFIG` environment variable is now checked for retrieving credentials from the docker config. ([#1618](https://github.com/GoogleContainerTools/jib/issues/1618))
```

Comment:
How about elaborate on this to message that `DOCKER_CONFIG` should be a directory containing a Docker config file?